core/src/main/java/hudson/security/HudsonAuthenticationEntryPoint.java

```diff
@@ -102,6 +102,7 @@ public void commence(ServletRequest request, ServletResponse response, Authentic
             }
             out.printf(
                 "<html><head>" +
+                "<meta http-equiv='Content-Type' content='text/html;charset=utf-8'>" +
                 "<meta http-equiv='refresh' content='1;url=%1$s'/>" +
```

Comment:
This does not seem to do what it claims to in the issue, as it sets the encoding for the HTTP redirect page, not the `/login` URL. Has this change been tested at all? What am I missing?